hive/src/test/java/com/netflix/iceberg/hive/ScriptRunner.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * Slightly modified version of the com.ibatis.common.jdbc.ScriptRunner class
```

Comment:
I think this should go below the copyright header. We will also need to add a section to LICENSE for it. I can add it after this PR is merged.

Reply:
Moved.